scripts/update.sh

```diff
@@ -44,10 +44,18 @@ cd $ROOT_DIR
 # Download docker images from docker-compose.yml.
 # As this might be run during setup we use `newgrp` command to ensure 
 # docker works.
+set +e
 newgrp docker << END
-# You can do more lines than just this./
 docker compose pull
 END
+# Check if the docker images download failed
+# In case of failure, the images will be built locally
+if [ $? -ne 0 ]; then
+    newgrp docker << END
+    docker compose up --no-start
```

Comment:
Is this enough? Or should the build script be called? I believe that script has some additional steps, but I'm not 100% sure

Reply:
In the tests I did here, yes. The UP command is used to start the container and if the image does not exist, it builds one. The argument --no-start we give is that he didn't start the services at that time.

Reply:
Now we are using build scripts. Thanks for it :)